.drone.star

```diff
@@ -107,6 +107,7 @@ config = {
             },
             "visualTesting": True,
             "screenShots": True,
+            "earlyFail": True,
```

Comment:
You dont need to set this for every one. If you set this to True in default section, then you can only override ones where you need to set false.